src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+.header,
+.nav__list,
+.nav__link {
+  display: flex;
+}
+
+.header,
+.nav__list {
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header {
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background: #fff;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  &__list,
+  &__link {
+    display: flex;
+  }
+
+  &__list {
+    justify-content: space-between;
+    list-style: none;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    position: relative;
+    text-decoration: none;
```

Comment:
I saw, that you already set 'text-decoration: none' in reset.css. Are they equal?